### PR TITLE
tamzen: init at 1.11.4

### PR DESCRIPTION
**Pull request by wishfort36**
###### Motivation for this change

Tamsyn is a cool font, but it doesn't have a powerline version.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`) (none)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -8253,6 +8253,12 @@
     githubId = 6016963;
     name = "Patrick Winter";
   };
+  wishfort36 = {
+    email = "42300264+wishfort36@users.noreply.github.com";
+    github = "wishfort36";
+    githubId = 42300264;
+    name = "wishfort36";
+  };
```
Comment (jonringer) [inline maintainers/maintainer-list.nix:8261]: Please make this into a separate commit:

````
maintainers: add wishfort36
```

File: pkgs/data/fonts/tamzen/default.nix [hunk 2]
```diff
@@ -0,0 +1,49 @@
+{ fetchFromGitHub, fontforge, mkfontscale, stdenv }:
+
+let
+  version = "1.11.4";
+in stdenv.mkDerivation {
+  pname = "tamzen-font";
+  inherit version;
```
Comment (jonringer) [inline pkgs/data/fonts/tamzen/default.nix:7]: use recursive attr set
```suggestion
stdenv.mkDerivation rec {
  pname = "tamzen-font";
  version = "1.11.4";
```

**Review by jonringer — COMMENTED** [review]
most fonts also use fetchzip, as you don't really need a stdenv. Althought the otb creation may warrant otherwise

cc @rnhmjoj

**Comment by jonringer** [comment]
also, thanks for opening your first PR

**Review by rnhmjoj — CHANGES_REQUESTED** [review]
The package looks good. I still have to test the .otb fonts: fontforge doesn't always produce good results.

**Comment by rnhmjoj** [comment]
Tested the otb fonts in urxvt and they seem all right.

**Comment by wishfort36** [comment]
I've done the requested changes. Thanks for reviewing, I'm not well versed with Nix, so I at first packaged tamzen for it to just work.

> also, thanks for opening your first PR

Thanks! I'm trying NixOS and found that it didn't package the coolest font I know of, so I felt that I had to contribute.

**Comment by rnhmjoj** [comment]
Can you squash the subsequent changes to `tamzen` in a single commit?

If you have never done it before you can do it like this:
  - Run `git rebase -i HEAD~5`,
  - Then reorder the commits and use the `fixup` action
  - save, exit and fix the conflicts if necessary

You should keep two commits:
  1. `maintainers: add wishfort36`
  2. tamzen: init at 1.11.4

**Comment by wishfort36** [comment]
Done.

**Review by rnhmjoj — APPROVED** [review]

**Comment by rnhmjoj** [comment]
Thank you!
